akka-stream/src/main/scala/akka/stream/impl/fusing/Ops.scala

```diff
@@ -25,10 +25,33 @@ import akka.stream.impl.Stages.DefaultAttributes
 /**
  * INTERNAL API
  */
-final case class Map[In, Out](f: In ⇒ Out, decider: Supervision.Decider) extends PushStage[In, Out] {
-  override def onPush(elem: In, ctx: Context[Out]): SyncDirective = ctx.push(f(elem))
+// FIXME: Not final because InterpreterSupervisionSpec. Some better option is needed here
```

Comment:
I think so. Until I refactor the InterpreterSupervisionSpec. Might be tomorrow as part of cleanup.